Cargo.toml

```diff
@@ -77,11 +77,11 @@ tempfile = { version = "3.3.0", default-features = false }
 wat = { version = "1.0", default-features = false }
 
 # binary dependencies
-wasm_rust_tests  = { path = "tests/wasm/wasm_rust_tests", artifact = "bin", target = "wasm32-wasi", default-features = false }
+enarx_wasm_tests  = { path = "tests/crates/enarx_wasm_tests", artifact = "bin", target = "wasm32-wasi", default-features = false }
 [target.'cfg(all(target_os = "linux", target_arch = "x86_64"))'.dev-dependencies]
-rust_exec_tests = { path = "tests/rust_exec_tests", artifact = "bin", target = "x86_64-unknown-linux-musl", default-features = false }
-rust_syscall_tests = { path = "tests/rust_syscall_tests", artifact = "bin", target = "x86_64-unknown-none", default-features = false }
-sev_attestation = { path = "tests/sev_attestation", artifact = "bin", target = "x86_64-unknown-linux-musl", default-features = false }
+enarx_exec_tests = { path = "tests/crates/enarx_exec_tests", artifact = "bin", target = "x86_64-unknown-linux-musl", default-features = false }
+enarx_syscall_tests = { path = "tests/crates/enarx_syscall_tests", artifact = "bin", target = "x86_64-unknown-none", default-features = false }
```

Comment:
Also not sure about the value of `enarx_` prefix here

Reply:
well it's not `sallyport_syscall_tests` but `enarx_syscall_tests`

Reply:
The distinction is not clear - all syscalls go through Sallyport, right? Are these testing syscalls coming from a workload running within a keep?

Reply:
These syscall test binaries replace the `exec` (formerly known as wasmldr).